src/assets/styles/global.scss

```
@@ -76,6 +107,40 @@ a:hover {
     gap: 30px;
 }
 
+/* Медиазапрос для мобильных устройств (ширина экрана до 767px) */
+@media (max-width: 767px) {
+    .footer {
+        /*  стили для контейнера на мобильных устройствах */
+        flex-direction: column;
+        /* ориентация для элементов внутри контейнера */
+        align-items: center;
+        /* Выравнивание по центру */
+        text-align: center;
+        /* Выравнивание текста по центру */
+        padding: 20px 0;
+        /* Измените отступы по вашему усмотрению */
+    }
+
+    .footer__logo {
+        width: 30%;
+        /*  ширина логотипа на мобильных устройствах */
+        margin-bottom: auto;
+        /*  отступ снизу логотипа на мобильных устройствах */
+    }
+
+    .footer__authors {
+        width: 50%;
+        /*  доступная ширина на мобильных устройствах */
+        justify-content: center;
```

```suggestion
        flex-direction: column;
        gap: 20px;
        padding-left: 10%;
```
Подумала, что тут вертикально симпатичнее смотрится, не так тесно))